doc/releases/changelog-0.23.0.md

```diff
@@ -537,37 +552,40 @@ the `decimals` and `show_matrices` keywords are added. `qml.drawer.tape_text(tap
 
 <h3>Bug fixes</h3>
 
-* Fixes a bug where non-trainable arguments were shifted in the `NesterovMomentumOptimizer`
+* Fixed a bug arising when multiplying `Hamiltonian` objects.
+  [(#2466)](https://github.com/PennyLaneAI/pennylane/pull/2466)
```

Comment:
Not essential, but might be nice to briefly mention what the bug was.

Reply:
It can still be vague, e.g., like the one below:

> Fixed a bug in the qml.PauliRot operation, where computing the generator was not taking into account the operation wires. 

It just allows readers to work out if they might have been affected by this bug, and whether they should upgrade to fix it.